### PR TITLE
Implemented Social Share buttons

### PR DESCRIPTION
**Pull request by taleymirza**

### CODE REVIEW

**Comment by mashhoodr** [comment]
@taleymirza can you please rebase

File: client/src/containers/Organisation/Organisation.js [hunk 1]
```diff
@@ -16,26 +19,40 @@ class Organisation extends Component<Props> {
...
-    const { organisation } = this.props.organisationDetail;
+    const { organisation } = this.props.currentOrganisation;
     return (
       <div>
-        {organisation && !organisation[0].detail ? (
+        {!isEmpty(organisation) ? (
           <Container>
-            <CardImg top width="100%" src={organisation[0].logo} />
-            <ContentHeader heading={organisation[0].name} />
+            <MetaTags>
```
Comment (mashhoodr) [inline client/src/containers/Organisation/Organisation.js]: @taleymirza @JavaEria this can be a component on its own.

Comment (taleymirza) [reply client/src/containers/Organisation/Organisation.js]: @mashhoodr should we create another branch to make it a separate component?

Comment (mashhoodr) [reply client/src/containers/Organisation/Organisation.js]: yes

**Pull request merged**